### PR TITLE
Use "adaptive" interactor as default.

### LINKED ISSUE #182: Using `extension_loaded` when building the container is not the right approach
I've spent a bit of time debugging why the bundle wouldn't work with our Heroku-based setup. It turns out that it's because it's using the `extension_loaded('newrelic')` condition when building the cache to know to use either the "blackhole" or the "real" implementation. 

The reason this happened? The container is built during the Heroku build phase. Because the extension is only enabled after, this `extension_loaded` end up being false and it silently uses the blackhole configuration.

## Next steps?

1. The Blackhole should log a warning (at least debug) when doing so, because there is no way to know that it used the blackhole one (`logging: true` still logs that it "does the job" because it's a decorator).
2. Uses `extension_loaded` within an implementation of `NewRelicInteractorInterface` that could be called `TolerateNoPhpExtensionInterfactor` that would either route to the blackhole or the "real" one.

WDYT?

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix #182

### CODE REVIEW

File: DependencyInjection/EkinoNewRelicExtension.php [hunk 1]
```diff
@@ -145,12 +146,17 @@ private function getInteractorServiceId(array $config): string
             return BlackholeInteractor::class;
         }
 
+        if ('auto' === $config['enabled']) {
```
Comment (jderusse) [inline DependencyInjection/EkinoNewRelicExtension.php]: `$config['interactor']`

Comment (Nyholm) [reply DependencyInjection/EkinoNewRelicExtension.php]: Good catch!

**Comment by Nyholm** [comment]
I added some change log. Are you okey with a beta3 after this is merged?

**Comment by jderusse** [comment]
:+1:

**Comment by sroze** [comment]
Thank you @Nyholm.

**Pull request merged**